rancid-tomatillos/src/App.js

```diff
@@ -1,26 +1,32 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, {Component} from "react";
+import MainView from "./components/MainView";
+import CardsContainer from "./components/CardsContainer";
```

Comment:
Just noting that we talked about file organization - this makes sense for now, as a small-scale project to just have 1 components directory holding all the comps we use